transmart-rest-api/src/main/groovy/org/transmartproject/rest/serialization/HypercubeJsonSerializer.groovy

```diff
@@ -200,6 +159,52 @@ class HypercubeJsonSerializer extends HypercubeSerializer {
         cell
     }
 
+    protected void writeValue(Object value) {
+        if (value == null) {
+            writer.nullValue()
+        } else if (value instanceof String) {
+            writer.value(value as String)
+        } else if (value instanceof Number) {
+            writer.value(value as Double)
+        } else if (value instanceof Date) {
+            def time = Instant.ofEpochMilli((value as Date).time).toString()
+            writer.value(time)
+        } else if (value instanceof Map) {
+            def obj = value as Map
+            writer.beginObject()
+            obj.each { k, v ->
```

Comment:
@gijskant Closure calls can be quite slow, and in this form (probably) will be. In CompileStatic code you should use a for(Entry e : obj) { ... } loop.